dpimport/__init__.py

```diff
@@ -4,6 +4,7 @@
 import logging
 import mimetypes as mt
 from . import patterns
+from dppylib import diff_files
```

Comment:
Where is the new import `diff_files` used in this file?

Reply:
Previously it was called without being defined:

https://github.com/PREDICT-DPACC/dpimport/blob/f4c72781733383f623dbcfea990451a1df3560b1/dpimport/__init__.py#L98